man/Makefile

```diff
@@ -36,6 +36,7 @@ man%:
 .PHONY: install
 install: all
 	for sec in $(sections); do \
```

Comment:
Oh, it occurs to me that we should probably include `set -e` here too because that's only the default in POSIX compliant mode, apparently? https://www.gnu.org/software/make/manual/html_node/Choosing-the-Shell.html

Reply:
Oh! Yes I recall that indeed; I recall I had a PR once to consider explicitly enabling posix mode for that reason;

- https://github.com/moby/moby/pull/41741